lti_consumer/tests/test_utils.py

```diff
@@ -0,0 +1,96 @@
+"""
```

Comment:
This was a move from `lti_consumer/tests/unit/test_utils.py` to `lti_consumer/test/test_utils.py`. This was to accommodate a file in `lti_consumer/test/unit` named `test_utils.py`, which contains actual tests of this module of testing utilities. This should be displayed as a renamed file, but it's not.